Livro.java

```diff
@@ -0,0 +1,103 @@
+import java.time.LocalDate;
+import org.junit.jupiter.api.Assertions;
+
+public class Livro {
+	
+	private String titulo;
+	private String resumo;
+	private String sumario;
+	private float preco;
+	private int paginas;
+	private String isbn;
+	private LocalDate dataPublicacao;
+	private Categoria categoria;
+	
+	
+	public Livro (String titulo, String resumo, String sumario, float preco,
+				  int paginas, String isbn, LocalDate data, Categoria categoria) {
+		
+		Assertions.assertNotNull(titulo,"Título inválido");
```

Comment:
O problema das Assertions do Junit é que elas soltam AssertionFailedError, que é uma exception não imaginada para falha de argumentos. Isso pode gerar confusão em que for avaliar o problema,  já que a pessoa pode demorar de entender que estamos o junit fora do contexto de testes.

Reply:
Qual Assert eu deveria incluir?